crates/bevy_ecs/src/system/system_piping.rs

```diff
@@ -232,6 +233,54 @@ pub mod adapter {
         res.unwrap()
     }
 
+    /// System adapter that utilizes the info! macro to print system information.
+    /// 
+    /// # Examples
+    /// 
+    /// ```
+    /// 
+    /// ```
+    pub fn info<T: Display>(In(data): In<T>) {
+        tracing::info!("{}", data);
+    }
```

Comment:
Is there a reason that this takes `T: Display` but others take `T: Debug`?

Reply:
Yep, I'd prefer `Debug` in general.